system-addon/lib/SnippetsFeed.jsm

```diff
@@ -212,7 +212,7 @@ this.SnippetsFeed = class SnippetsFeed {
         this._clearBlockList();
         break;
       case at.TOTAL_BOOKMARKS_REQUEST:
-        this.getTotalBookmarksCount(action._target.browser);
+        this.getTotalBookmarksCount(action.meta.fromTarget);
```

Comment:
hm, I wonder if we can figure out a test to prevent this error in the future...

Reply:
mochitests? 😖